public/manifest.json

```
@@ -0,0 +1,511 @@
+{
+  "name": "Nova",
+  "theme_color": "#000000",
+  "start_url": "/",
+  "background_color": "#000000",
+  "short_name": "Nova",
+  "description": "Nova is a JavaScript and WebAssembly engine written in Rust and following data-oriented design principles",
+  "display_override": [
+    "window-controls-overlay",
+    "standalone",
+    "fullscreen",
+    "browser",
+    "minimal-ui"
+  ],
+  "shortcuts": [
+    {
+      "name": "HomePage",
```

```suggestion
      "name": "Home",
```
**Nitpick:** Sounds better if it's displayed anywhere. But perhaps this is default for the home-page of a PWA?